src/components/QuoteBubble/CopyButton.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { CopyCards } from '../Icons';
+
+const Wrapper = styled.button`
+  position: absolute;
+  top: 6rem;
+  right: 0;
+`;
+
+function CopyButton({ onCopy = () => {} }) {
```

Comment:
Rather set **PropTypes.func.isRequired** in prop types instead of passing empty function. Since **CopyButton** component name even implies it has copy functionality it doesn't makes sense to even exist without this handler passed.

Reply:
Well imo, you should always provide default `noop` for every component handler. It is not about the use case, but about component in further development. Without `noop`, you need to always create some void function to render this component, which is inefficient during development. Also noop does not affect the code much and does not have any performance issues. I treat it as a good practices.